test/utils/actions.ts

```diff
@@ -161,9 +167,57 @@ export const App = {
       this.wait
     );
   },
+  openServiceWorkerTab: async function (this: mocha.Context) {
+    await this.driver.switchTo().newWindow('extension');
+    const extensionPanelHandle = await this.driver.getWindowHandle();
+
+    await this.driver.get(this.extensionPanel);
+
+    const extensionsManager = await (
+      await this.driver.findElement(By.css('extensions-manager'))
+    ).getShadowRoot();
+
+    const extensionsToolbar = await (
+      await extensionsManager.findElement(By.css('extensions-toolbar'))
+    ).getShadowRoot();
+    const devMode = await extensionsToolbar.findElement(By.id('devMode'));
+    const isDevMode = await devMode.getAttribute('checked');
+    if (!isDevMode) {
+      await devMode.click();
+    }
+
+    const extensionsDetailView = await (
+      await extensionsManager.findElement(By.css('extensions-detail-view'))
+    ).getShadowRoot();
+    const bg = await extensionsDetailView.findElements(
+      By.css('a.inspectable-view')
+    );
+    await bg[bg.length - 1].click();
+
+    const handles = await this.driver.getAllWindowHandles();
+    this.serviceWorkerTab = handles[handles.length - 1];
+
+    await this.driver.switchTo().window(extensionPanelHandle);
+    await this.driver.close();
+    await this.driver.switchTo().window(handles[0]);
+  },
+  restartServiceWorker: async function (this: mocha.Context) {
+    const currentHandle = await this.driver.getWindowHandle();
+
+    await this.driver.switchTo().newWindow('serviceworker');
```

Comment:
This method accepts only `tab` or `window` value.

https://www.selenium.dev/documentation/webdriver/browser/windows/#create-new-window-or-new-tab-and-switch